app/views/reports/regions/_patient_breakdown_charts.html.erb

```diff
@@ -1,13 +1,94 @@
 <div class="d-lg-flex flex-lg-wrap">
   <div class="d-lg-flex w-lg-50 pr-lg-2">
+    <div id="patient-breakdown" data-period="<%= @period.to_s %>" class="mt-8px mx-0px mb-16px p-20px bg-white br-4px bs-small w-lg-full mt-lg-0 pb-inside-avoid b-print-black w-print-16cm">
+      <div class="d-flex flex-1 mb-8px">
+        <h3 class="mb-0px mr-8px">
+          Hypertension patients
+        </h3>
+      </div>
+      <p class="mb-16px c-grey-dark mr-lg-12px">
+        Hypertension patient coverage and breakdown in <%= @region.name %>
+      </p>
+      <% if @region.district_region? || @region.state_region? %>
```

Comment:
Lets add a method on `region` to tell us if the particular region supports htn population figures...then all the places you are checking district || state region can use that method.  Something like `region.supports_htn_population_coverage?`